x-pack/plugins/stack_connectors/public/connector_types/security/tines/translations.ts

```diff
@@ -0,0 +1,262 @@
+    defaultMessage: 'Email',
+  }
+);
+export const TOKEN_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.config.tokenTextFieldLabel',
+  {
+    defaultMessage: 'API token',
+  }
+);
+
+export const URL_INVALID = i18n.translate(
+  'xpack.stackConnectors.security.tines.config.error.invalidUrlTextField',
+  {
+    defaultMessage: 'Tenant URL is invalid.',
+  }
+);
+
+export const EMAIL_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.security.tines.config.error.requiredEmailText',
+  {
+    defaultMessage: 'Email is required.',
+  }
+);
+export const TOKEN_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.security.tines.config.error.requiredAuthTokenText',
+  {
+    defaultMessage: 'Auth token is required.',
+  }
+);
+
+// params form
+export const STORY_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.storyFieldLabel',
+  {
+    defaultMessage: 'Tines Story',
+  }
+);
+export const STORY_HELP = i18n.translate('xpack.stackConnectors.security.tines.params.storyHelp', {
+  defaultMessage: 'The Tines story to send the events to',
+});
+export const STORY_PLACEHOLDER = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.storyPlaceholder',
+  {
+    defaultMessage: 'Select a story',
+  }
+);
+export const STORY_ARIA_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.storyFieldAriaLabel',
+  {
+    defaultMessage: 'Select a Tines story',
+  }
+);
+export const STORY_PUBLISHED_BADGE_TEXT = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.storyPublishedBadgeText',
+  {
+    defaultMessage: 'Published',
+  }
+);
+
+export const WEBHOOK_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookFieldLabel',
+  {
+    defaultMessage: 'Tines Webhook action',
+  }
+);
+export const WEBHOOK_HELP = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookHelp',
+  {
+    defaultMessage: 'The data entry action in the story',
+  }
+);
+export const WEBHOOK_PLACEHOLDER = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookPlaceholder',
+  {
+    defaultMessage: 'Select a webhook action',
+  }
+);
+export const WEBHOOK_DISABLED_PLACEHOLDER = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookDisabledPlaceholder',
+  {
+    defaultMessage: 'Select a story first',
+  }
+);
+export const WEBHOOK_ARIA_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookFieldAriaLabel',
+  {
+    defaultMessage: 'Select a Tines webhook action',
+  }
+);
+
+export const WEBHOOK_URL_LABEL = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookUrlFieldLabel',
+  {
+    defaultMessage: 'Webhook URL',
+  }
+);
+export const WEBHOOK_URL_FALLBACK_TITLE = i18n.translate(
+  'xpack.stackConnectors.security.tines.params.webhookUrlFallbackTitle',
+  {
+    defaultMessage: 'Tines API results limit reached',
+  }
+);
+export const WEBHOOK_URL_FALLBACK_TEXT = (type: 'Story' | 'Webhook') =>
```

Comment:
I think typescript will be angry for using `type` because its a reserved word
```suggestion
export const WEBHOOK_URL_FALLBACK_TEXT = (tinesType: 'Story' | 'Webhook') =>
```